packages/fast-components-class-name-contracts-msft/src/call-to-action/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * The class name contract for the call to action component
+ */
+export interface ICallToActionClassNameContract {
+    callToAction: string;
+    callToAction_backer: string;
```

Comment:
lets change this to backplate to be more semantic

Reply:
Can we align these with the actual appearance type? This is a subset of button, so I would assume that if I wanted to override these changes I would do so by targeting the appearance, similar to how we have it in Button.

callToAction_primary: string;
callToAction_secondary: string;